app/templates/components/events/import-event.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="ui stackable centered grid">
+  <div class="twelve wide column">
+    <h3>Import Event</h3>
+    <hr class="hr-line">
+    <p class="section-heading">Select event source file to import</p>
+    <p><b>Supported formats:</b> Open Event compatible json package (.zip), PentabarfXML (.xml), iCalendar (.ical)</p>
+
+    <input type="file" name="file">
+
+    <div class="create-event">
+      <button class="ui primary button">
+        Create Event
```

Comment:
Let's call it Import event. And use translation helper. 
`{{t 'Import Event'}}`